src/fparser/tests/test_splitline.py

```diff
@@ -0,0 +1,139 @@
+# Modified work Copyright (c) 2017 Science and Technology Facilities Council
```

Comment:
New test file containing tests previously included in splitline source (and therefore presumably not run by py.test). Great.